jgiven-html5-report/src/main/java/com/tngtech/jgiven/report/html5/Html5AttachmentGenerator.java

```diff
@@ -115,4 +125,41 @@ private File writeFile( AttachmentModel attachment, MediaType mediaType ) {
         return targetFile;
     }
 
+    private String compressToThumbnail( String base64content, String extension ) {
+        byte[] imageBytes = javax.xml.bind.DatatypeConverter.parseBase64Binary(base64content);
```

Comment:
I would extract a method here that takes a BufferedImage and produces one and do the Base64 conversion outside of that.